test/s3-benchmarks/pom.xml

```diff
@@ -54,12 +54,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>s3-transfer-manager</artifactId>
-            <version>${awsjavasdk.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>software.amazon.awssdk.crt</groupId>
-            <artifactId>aws-crt</artifactId>
-            <version>${awscrt.version}</version>
+            <version>${awsjavasdk.version}-PREVIEW</version>
```

Comment:
Same here, we need to take mainline change

Reply:
fixed version